sdks/python/src/honcho/peer.py

```
@@ -112,7 +112,9 @@ def chat(
             return None
         return response.content
 
-    def get_sessions(self) -> SyncPage[Session]:
+    def get_sessions(
+        self, filter: dict[str, object] | None = None
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)